client/state/ui/README.md

```diff
@@ -2,39 +2,3 @@ UI State
 ========
 
 A module for managing user interface state.
-
-## Actions
```

Comment:
In the end, I decided to keep the docs and updated them to include the user offerings I had neglected to include in #2080 (273de83, 06044a9).